### ISSUE TITLE
"Passthrough is not supported, GL is disabled, ANGLE is" reducing audio quality

### ISSUE THREAD

**Issue by DarwinChing** [w-okada/voice-changer#492]
### Issue Type

Bug Report, Question

### vc client version number

MMVCServerSIO_win_onnxgpu-cuda_v.1.5.3.9a

### OS

Windows 11

### GPU

Nvidia GTX 1080

### Clear setting

yes

### Sample model

no

### Input chunk num

yes

### Wait for a while

The GUI successfully launched.

### read tutorial

yes

### Extract files to a new folder.

yes

### Voice Changer type

RVC

### Model type

pyTorch, ONNX

### Situation

Basically shortly after launching the program, and starting a voice over, the error :"Passthrough is not supported, GL is disabled, ANGLE is" reducing audio quality" shows up and then audio quality becomes drastically worse, very robotic.

This occurs across different versions of the program.

**Comment by DarwinChing** [comment]
Update: I updated my drivers and it still occurs.

**Comment by w-okada** [comment]
"Passthrough is not supported, GL is disabled, ANGLE is" have no effect. read tutorial.

**Comment by w-okada** [comment]
Does the same thing happen when you run it on the CPU?
